demo_drivers/meshing/Makefile.am

```diff
@@ -8,6 +8,7 @@ mesh_from_xfig_triangle \
 mesh_from_vmtk \
 quad_from_triangle_mesh \
 quadratic_vmtk_tetgen \
+mesh_from_another_mesh_poisson \
```

Comment:
triangle_mesh_from_another_triangle_mesh. Can save some space (just in the directory name) by omitting poisson.